### DISCUSSION TITLE
be unable to authenticate as a student

### DISCUSSION THREAD

**Discussion by firstsm-41** [community/community#138338] [GitHub Education]
### Hi everyone,

Unable to authenticate students.

Even if you get a certificate of enrollment in English, it will not be certified
![스크린샷 2024-09-11 095952](https://github.com/user-attachments/assets/f3cb2105-abe6-4919-a526-67a3521c10ff)
![스크린샷 2024-09-11 100002](https://github.com/user-attachments/assets/7701108d-c7a2-47c4-90f0-3ba13fc12dec)
![스크린샷 2024-09-11 100009](https://github.com/user-attachments/assets/95cee538-356b-4d8a-866c-6fb964c7b772)
![스크린샷 2024-09-11 100016](https://github.com/user-attachments/assets/fb07ecc1-bfa6-4258-acef-95a143fd42c5)

**Comment by Davdadev** [comment]
use your student id 
![download](https://github.com/user-attachments/assets/05c63ee9-d08d-4443-9465-d18484ac5a31)

**Reply by firstsm-41** [reply]
There is no English on my student ID, only Korean. So I submitted my English certificate of enrollment, but can I submit my Korean student ID?

**Comment by Davdadev** [comment]
[Unclear academic affiliation documents](https://docs.github.com/en/education/explore-the-benefits-of-teaching-and-learning-with-github-education/github-education-for-students/why-wasnt-my-application-to-github-education-for-students-approved#unclear-academic-affiliation-documents)
If the dates or schedule mentioned in your uploaded image do not match our eligibility criteria, we require further proof of your academic status.

If the image you uploaded doesn't clearly identify your current academic status or if the uploaded image is blurry, we require further proof of your academic status. When you reapply, upload an image of your school ID, academic transcript, or other document showing proof of your academic status that clearly shows at least one date.

Note: PDF files are not supported and you cannot upload them as proof of affiliation.

**Reply by firstsm-41** [reply]
Then I can attach one file, but do I not have to attach my enrollment certificate?
